athenz/resource_role_test.go

```diff
@@ -757,7 +761,7 @@ resource "athenz_role" "roleTest" {
 `, name, domain, member1)
 }
 
-func testAccGroupRoleConfigAddMemberWithExpiration(name, domain, member1, member2 string) string {
+func testAccGroupRoleConfigAddMemberWithExpirationAndReview(name, domain, member1, member2 string) string {
```

Comment:
is it possible to have a review without expiration?
why to  test both together?

Reply:
Done,  now its seperated